src/main/scala/nl.knaw.dans.easy.sword2/DepositProperties.scala

```diff
@@ -119,6 +121,8 @@ class DepositProperties(depositId: String, depositorId: Option[String] = None)(i
   def getLastModifiedTimestamp: Option[FileTime] = {
     modified
   }
+
+
```

Comment:
Why these extra new lines? 
Also please import https://github.com/DANS-KNAW/easy-module-archetype/blob/master/dans-intellij-codestyles.xml in your intellij _preferences - editor - code-style_ and apply the code formatting

Reply:
👍 , but we checked this and it turns out our style does not remove 2 extra lines before a brace. Maybe we should change the style.